src/opnsense/www/js/widgets/IpsecTunnels.js

```diff
@@ -29,7 +30,7 @@ import BaseTableWidget from "./BaseTableWidget.js";
 export default class IpsecTunnels extends BaseTableWidget {
     constructor() {
         super();
-        this.currentTunnels = {};
+        this.resizeHandles = "e, w";
```

Comment:
I think this has to go again since it was removed here earliee today:

https://github.com/opnsense/core/commit/4bd6ad3cdec0ad2eb2dc4896c85bd9844f9e4f8e